### DISCUSSION TITLE
F3 Entity Bug

### DISCUSSION THREAD

**Discussion by ToastyLux** [FabricMC/fabric-api#3171] [Player support]
### Description

Im on 1.16.1 with these mods: atum, dynamic menu, lazydfu, lazystronghold, lithium, sodium speedrun build, spedrunigt, standard settings, starlight.

Issue: When I open the f3 menu, my E or Entity count is -1.

Does anyone know how to fix this? 
![2023-07-03_17 41 22](https://github.com/FabricMC/fabric/assets/84648930/4ace4e0a-1e92-4dd5-bada-2bdd6a42753f)

**Comment by maximedelis** [comment]
Got the same issue. Found a fix? My idea's that it was related to using either Integrated vs. Dedicated GPU (this bugged happened when I switch to dedicated).

EDIT: Alright that was actually Entity Culling being turned on....

**Comment by tully314159-jpg** [comment]
yeah just turn off entity culling, ive had same issue

**Reply by ToastyLux** [reply]
yes
